charts/gardener/controlplane/charts/application/templates/rbac-user.yaml

```
@@ -241,9 +190,10 @@ rules:
 apiVersion: {{ include "rbacversion" . }}
 kind: ClusterRole
 metadata:
-  name: garden.sapcloud.io:system:project-viewer
+  name: gardener.cloud:system:project-viewer
```

```suggestion
  name: gardener.cloud:system:project-viewer-aggregation
```
This ClusterRole is duplicated now: see line 209 https://github.com/gardener/gardener/pull/1601/files#diff-db48d4f9428af05632229707112cf2b3R209